src/__tests__/PojoMap.test.ts

```diff
@@ -14,4 +14,54 @@ describe('PojoMap', () => {
   it('should make an empty Map', () => {
     expect(PojoMap.empty<string, number>()).toEqual({});
   });
+
+  it('should get a value via its key', () => {
```

Comment:
Remove references to values here, this is just a key existence check
```suggestion
  it('should check whether it is defined at a key', () => {
```